packages/core/src/card/card.element.scss

```diff
@@ -0,0 +1,14 @@
+@import './../styles/tokens/generated/index';
+
+:host {
+  --width: auto;
+  --height: auto;
```

Comment:
After testing in storybook I think `--height` and `--width` need to be `100%` when set to auto they don't fill their container so multi grid/column layouts don't work. Example:

```html
<section cds-layout="grid gap:lg cols@md:6">
  <cds-card  style="--height: 100%">
    <div cds-layout="vertical gap:md">
      <h3 cds-text="section">Card Title</h3>

      <div cds-layout="vertical gap:lg p-y:md">
        <p cds-text="body">
          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.  
        </p>

        <p cds-text="body">
          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.  
        </p>
      </div>
    </div>
  </cds-card>

  <cds-card style="--height: 100%">
    <div cds-layout="vertical gap:md">
      <h3 cds-text="section">Card Title</h3>

      <p cds-text="body" cds-layout="p-y:md">
        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.  
      </p>
    </div>
  </cds-card>
</section>
```

Also it would be nice to see a column layout demo like this on the mdx page to show a bit more with card layouts.

Reply:
This is what we wanted in the first place. 

The panel to be able to fit the container, the card to be able to resize depending on its content.

Reply:
Yeah you're right, I'll try something else see if the cards can fill without adjusting the --height 👍

Reply:
There will be a CSS after all. If you expect every card to take the full width and height of a container, you will be able to simply do 

```
cds-card {
 --height: 100%;
 --width: 100%;
}
```